src/app/Main/Header/Export/Header-ExportContainer.component.tsx

```diff
@@ -58,14 +58,13 @@ export function HeaderExportContainerComponent(): JSX.Element {
             `QueryPage:Filters.${data.column}`
         );
     } else {
-        setting.tableAttributeNames.column = undefined
+        setting.tableAttributeNames.column = undefined;
     }
 
-    const filterSettingsPath = history.location.search.replace(/&row.+/g, "").replace(/&column.+/g, "")
-
-    const isolateFilteredUrl: string = ISOLATE_URL + filterSettingsPath;
-
-
+    const urlParams = new URLSearchParams(history.location.search);
+    urlParams.delete("row");
+    urlParams.delete("column");
+    const isolateFilteredUrl = `${ISOLATE_URL}/${urlParams}`;

```

Comment:
In general, i try to be as explicit as possible, so i would use: `urlParams.toString()` to make clear that some conversion from a params object to string happens here. Most developers wouldn't use that because in most languages toString() is defined on object (and therefore present on all classes) and called by default if used in a string expression.
So another reason i prefer the explicit way is to remind me of implementing toString() on my own classes :)